lab-tara/app/src/main/java/com/example/filterablelist/Friends/Friend.java

```diff
@@ -0,0 +1,29 @@
+package com.example.filterablelist.Friends;
+
+import android.content.Intent;
+
+public class Friend {
+    public String name;
+    public String handle;
+    public String bio;
+
+    public Friend(String name, String handle, String bio) {
+        this.name = name;
+        this.handle = handle;
+        this.bio = bio;
+    }
+
+    public void fillIntent(Intent intent) {
```

Comment:
This is a perfectly reasonable approach in this case since the data is small. The other approach that is often taken, especially when your model starts to get bigger, is to pass some sort of id in the intent rather than all the data, and have the receiver just look up the correct record. No need to change this, just something to keep in mind.